package.json

```diff
@@ -14,14 +14,18 @@
     "test:unit:verbose": "jest --coverage --no-cache",
     "test:unit": "yarn test:unit:verbose --silent",
     "test:unit:watch": "yarn test:unit:output -- --watch",
-    "test:unit:output": "jest --json --outputFile=jest-test-results.json --no-cache --silent",
+    "test:unit:output":
+      "jest --json --outputFile=jest-test-results.json --no-cache --silent",
     "prepush": "yarn test",
     "start": "yarn test:unit:output && start-storybook -p 6006",
     "prebuild": "yarn test:unit:output",
-    "build": "yarn build:stylesheet && yarn build:global-styles && yarn build:copy-global-styles && yarn build:storybook",
+    "build":
+      "yarn build:stylesheet && yarn build:global-styles && yarn build:copy-global-styles && yarn build:storybook",
     "build:storybook": "build-storybook -c .storybook -o .out",
-    "build:stylesheet": "mkdir -p dist && BABEL_ENV=static node build/stylesheet > dist/circuit-ui.css",
-    "build:global-styles": "mkdir -p dist && ./build/global-styles > dist/circuit-ui-global.css",
+    "build:stylesheet":
+      "mkdir -p dist && BABEL_ENV=static node build/stylesheet > dist/circuit-ui.css",
+    "build:global-styles":
+      "mkdir -p dist && ./build/global-styles > dist/circuit-ui-global.css",
```

Comment:
Prettier shouldn't format this file (it's blacklisted in .prettierignore).

Reply:
Maybe someone ran "format document" in VSCode?

Reply:
Its have it on speed dial :)

Reply:
Commit is dropped.